### PR TITLE
Add rename permission

### PR DESCRIPTION
**Pull request by 1-alex98**
For admins or global moderators.
Tested the migration.

### CODE REVIEW

**Comment by 1-alex98** [comment]
```
Message    : Field 'id' doesn't have a default value
```
Why did it work on the tets server then? (I cleaned the test server up afterwards)

**Comment by bukajsytlos** [comment]
I think group permissions are now managed via repeatable migrations in R__050_group_permission.sql

**Review by bukajsytlos — CHANGES_REQUESTED** [review]
Add it to R__050_group_permission.sql

**Comment by 1-alex98** [comment]
Repeatable migrations ???

**Comment by 1-alex98** [comment]
Execute after migration on dev/prod:
```
INSERT INTO group_permission_assignment (group_id, permission_id)
SELECT user_group.id, group_permission.id FROM user_group, group_permission
WHERE (user_group.technical_name = 'faf_moderators_global')
  AND group_permission.technical_name IN ('ROLE_ADMIN_ACCOUNT_NAME_CHANGE');

-- Server admins
INSERT INTO group_permission_assignment (group_id, permission_id)
SELECT user_group.id, group_permission.id FROM user_group, group_permission
WHERE user_group.technical_name = 'faf_server_administrators'
  AND group_permission.technical_name IN ('ROLE_ADMIN_ACCOUNT_NAME_CHANGE');

```

**Review by bukajsytlos — APPROVED** [review]
lgtm

File: migrations/R__050_group_permission.sql [hunk 1]
```diff
@@ -28,7 +28,8 @@ INSERT INTO `group_permission` (id, technical_name, name_key) VALUES
 (21,'WRITE_MESSAGE','permission_group.write_message'),
 (22,'ADMIN_KICK_SERVER','permission_group.admin_kick_server'),
 (23,'ADMIN_BROADCAST_MESSAGE','permission_group.admin_broadcast_message'),
-(24,'ADMIN_JOIN_CHANNEL','permission_group.admin_join_channel')
+(24,'ADMIN_JOIN_CHANNEL','permission_group.admin_join_channel'),
+(25,'ROLE_ADMIN_ACCOUNT_NAME_CHANGE', 'permission_group.admin_name_change')
```
Comment (Brutus5000) [inline migrations/R__050_group_permission.sql]: Remove the ROLE_ prefix, that's a Spring internal thingy.
Name key misses _account part

**Review by Brutus5000 — CHANGES_REQUESTED** [review]

**Review by Brutus5000 — APPROVED** [review]

**Pull request merged**